sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
 
 ### Features Added
 
+- Perform CRUD Operations for Application Gateway for Containers (AGC) Security Policy
+- Use the latest API version for (2025-01-01)
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
+
+## 1.1.0-beta.1 (Unreleased)
```

Comment:
please add your change as an item in the existing "unreleased" version entry. We should only have one "Unreleased" entry